packages/zoe/src/typeGuards.js

```diff
@@ -127,7 +127,7 @@ export const InvitationElementShape = M.splitRecord({
 });
 
 export const OfferHandlerI = M.interface('OfferHandler', {
-  handle: M.call(SeatShape).optional(M.any()).returns(M.string()),
+  handle: M.call(SeatShape).optional(M.any()).returns(M.any()),
```

Comment:
that could be a separate fix

Reply:
https://github.com/Agoric/agoric-sdk/pull/8748